app/Listeners/PluginError/RecordPluginErrorInBugsnag.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+
+namespace App\Listeners\PluginError;
+
+use App\Events\PluginErrorReceivedEvent;
+use App\Exceptions\PluginErrorException;
+use Bugsnag\BugsnagLaravel\Facades\Bugsnag;
+
+class RecordPluginErrorInBugsnag
+{
+    /**
+     * Handle any squawk allocation event
+     *
+     * @param PluginErrorReceivedEvent $errorReceivedEvent
+     * @return bool
+     */
+    public function handle(PluginErrorReceivedEvent $errorReceivedEvent) : bool
+    {
+        Bugsnag::notifyException(new PluginErrorException('New plugin error reported'));
```

Comment:
This isn't required; Bugsnag will catch any exception that bubbles up to the `ExceptionHandler` class.